tests/suites/test_suite_pkcs7.function

```diff
@@ -0,0 +1,405 @@
+/* BEGIN_HEADER */
+#include "mbedtls/bignum.h"
+#include "mbedtls/pkcs7.h"
+#include "mbedtls/x509.h"
+#include "mbedtls/x509_crt.h"
+#include "mbedtls/x509_crl.h"
+#include "mbedtls/oid.h"
+#include "sys/types.h"
+#include "sys/stat.h"
+#include "unistd.h"
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE */
```

Comment:
```suggestion
/* BEGIN_CASE depends_on:MBEDTLS_FS_IO */
```
and likewise for each test function that loads data from files.

Please minimize the use of external files in unit tests, because they can't run on platforms without stdio.

To check that the required dependencies are in place:
```
tests/scripts/all.sh -k test_no_pem_no_fs
```